arwen/cryptoapi/cryptoei.go

```diff
@@ -63,9 +70,45 @@ func CryptoImports(imports *wasmer.Imports) (*wasmer.Imports, error) {
 		return nil, err
 	}
 
+	imports, err = imports.Append("addEC", addEC, C.addEC)
+	if err != nil {
+		return nil, err
+	}
+
+	imports, err = imports.Append("doubleEC", doubleEC, C.doubleEC)
+	if err != nil {
+		return nil, err
+	}
+
+	imports, err = imports.Append("isOnCurveEC", isOnCurveEC, C.isOnCurveEC)
+	if err != nil {
+		return nil, err
+	}
+
+	imports, err = imports.Append("scalarBaseMultEC", scalarBaseMultEC, C.scalarBaseMultEC)
+	if err != nil {
+		return nil, err
+	}
+
+	imports, err = imports.Append("scalarMultEC", scalarMultEC, C.scalarMultEC)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
if you append these from a list by using a loop you can have that `	if err != nil {
		return nil, err
	}` only 1 time